cli/src/org/partiql/shell/Shell.kt

```diff
@@ -122,11 +123,17 @@ class Shell(
     }
 
     private fun run(exiting: AtomicBoolean) = TerminalBuilder.builder().build().use { terminal ->
-
+        val userSyntaxFile = homeDir.resolve(".nano/PartiQL.nanorc")
```

Comment:
We need to document this path if a customer wishes to use their own .nanorc file